core/src/main/groovy/noe/rhel/server/workspace/WorkspaceHttpdTomcatsBaseOS.groovy

```diff
@@ -71,10 +92,13 @@ class WorkspaceHttpdTomcatsBaseOS extends WorkspaceMultipleTomcats{
       serverController.getServerById(tomcatId).setHost(originalTomcatHosts.get(tomcatId))
     }
 
-    modJkModules.each() {File module ->
+    moduleFiles.each() { File module ->
+      log.debug("Removing file ${module}.")
```

Comment:
IIRC, JBFile has logging already, you don't have to put it here as well.

Reply:
This apply on all logging messages before JBFile

Reply:
Logging removed